crates/manifest/src/normalize.rs

```diff
@@ -103,3 +104,45 @@ fn normalize_trigger_ids(manifest: &mut AppManifest) {
         }
     }
 }
+
+fn apply_profile_overrides(manifest: &mut AppManifest, profile: Option<&str>) {
+    let Some(profile) = profile else {
+        return;
+    };
+
+    for (_, component) in &mut manifest.components {
+        let Some(overrides) = component.profile.get(profile) else {
+            continue;
+        };
```

Comment:
Should it be an error (or warning) to use a profile that doesn't appear in the manifest?

Reply:
Oh look there it is just below...